src/lib/isMobilePhone.js

```diff
@@ -95,6 +95,7 @@ const phones = {
   'fo-FO': /^(\+?298)?\s?\d{2}\s?\d{2}\s?\d{2}$/,
   'fr-BF': /^(\+226|0)[67]\d{7}$/,
   'fr-BJ': /^(\+229)\d{8}$/,
+  'fr-CD': /^(\+?243|0)?(8|9)\d{8}$/,
```

Comment:
```suggestion
  'fr-CD': /^(\+?243|0)?(8[0-2489]|9[017-9])\d{7}$/,
```
According to https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000370001PDFE.pdf with the addition of Africell phone numbers from https://github.com/google/libphonenumber/blob/master/resources/metadata/243/ranges.csv in line with the comment of the creator of the original PR

Reply:
We can fix this in a later PR as well